build-apisix-base.sh

```diff
@@ -173,3 +173,18 @@ cd ..
 cd amesh-${amesh_ver} || exit 1
 sudo OPENRESTY_PREFIX="$OR_PREFIX" sh -c 'PATH="${PATH}:/usr/local/go/bin" make install'
 cd ..
+
+# package etcdctl
+ETCD_ARCH="amd64"
+ETCD_VERSION=${ETCD_VERSION:-'3.5.4'}
```

Comment:
1、In general, when should we upgrade `etcdctl`?
2、Is there some correspondence between `etcdctl` and `etcd server` versions? What I want to say is, will `etcdctl` be unable to operate a certain version of `etcd server`, which will cause apisix to fail to work properly?

Reply:
I choose '3.5.4' because it is the version used in APISIX's CI.

1. since etcd will guarantee the compatibility in v3, and our use case with etcdctl is very simple, I think we don't need to upgrade it unless for security risk.
2. if etcd breaks the API, we are unable to operate a certain version of etcd server, whether we access it via HTTP or etcdctl or grpcurl doesn't bring any difference. We still need to update and blame etcd.